.github/workflows/simpletest-db-config.yml

```diff
@@ -0,0 +1,41 @@
+name: SimpleTest (Database Config)
+on:
+  pull_request:
+    branches: [1.*]
+jobs:
+  simpletest-db-config:
+    name: SimpleTest
+    runs-on: ubuntu-22.04
+    # Skip this job if the PR title contains "[skip tests]".
+    if: ${{ !contains(github.event.pull_request.title, '[skip tests]') }}
+    timeout-minutes: 20
+    strategy:
+      fail-fast: false
+      matrix:
+        php-versions: ['8.3']
```

Comment:
I guess, that's temporary (single php version)...?

Reply:
For the time being I was thinking we'd limit it to a single PHP version.